### DISCUSSION TITLE
Is there a way to download all videos of a channel after a given date

### DISCUSSION THREAD

**Discussion by Rido-o** [jmbannon/ytdl-sub#885] [Q&A] [answered]
As the title says, is there any way to download all of a channels videos after a specific given date, an example would be this [channel](https://www.youtube.com/@LEMMiNO/videos) after Jan 30, 2016.

Would something like:
```
date_range:
      after: "2016-30-01"
```
work?

Nevermind, found the [docs](https://ytdl-sub.readthedocs.io/en/latest/config_reference/plugins.html#date-range)

**Comment by jmbannon** [comment]
Glad you found it :)

**Reply by Rido-o** [reply, marked as answer]
ended up with something like this, that seems to work fine.

config:
```
  Youtube Archive:
    preset:
      - "video"
      - "sponsorblock_wait"
    date_range:
      after: "{start_date}"
    overrides:
      start_date: "20050423"
```

subscriptions:
```
  = Youtube | = TV-PG:
    "~LEMMiNO":
      url: "https://www.youtube.com/@LEMMiNO"
      start_date: "20160130"
```
